README.md

```diff
@@ -9,7 +9,7 @@ fancy paginator component
 
 ## Running unit tests
 
-PHPUnit 9 or newer is required.
+PHPUnit 10 or newer is required.
```

Comment:
This should be changed (it was wrong in the previous version too). We can't guarantee that tests will work with PHPUnit 11, so we need to remove the "or newer" clause.
If, instead, you manage to get the tests working with both, let's change the requirements to include version 11 too.

Reply:
Tests work with PHPUnit 10 and 11.
I have updated the README accordingly.

Reply:
> Tests work with PHPUnit 10 and 11. I have updated the README accordingly.

Forgive my scepticism, but I need to see the tests (and I can't, as long as we don't allow PHPUnit 11)

Reply:
Sorry, I forgot to mentioned that I have also allowed the installation of PHPUnit 11.

In PHP 8.1, tests are using PHPUnit 10, in higher PHP versions, PHPUnit 11 is used (see https://github.com/KnpLabs/knp-components/actions/runs/10227239091/job/28298482755?pr=340).